src/Akka.Persistence.Azure/Journal/HighestSequenceNrEntry.cs

```diff
@@ -1,71 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Microsoft.Azure.Cosmos.Table;
-
-namespace Akka.Persistence.Azure.Journal
-{
-    internal sealed class HighestSequenceNrEntry
```

Comment:
This class is a duplicate of `Akka.Persistence.Azure.TableEntities.HighestSequenceNrEntry`, deleting it so that it doesn't cause any bug in the future.